examples/job/job.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Job
+metadata:
+  name: test-job
+spec:
+  parallelism: 2
+  completions: 5
+  selector:
+    name: test-job
+  template:
+    metadata:
+      labels:
+        name: test-job
```

Comment:
Please don't use `name` as a label key. It confuses users.
https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/config-best-practices.md